exchange/exchange.go

```diff
@@ -104,7 +104,7 @@ type bidResponseWrapper struct {
 }
 
 type BidIDGenerator interface {
-	New() (string, error)
+	New(bidder string) (string, error)
```

Comment:
Generally I'm skeptical about modifying non-test code to enhance tests. It's probably ok in this case.

Reply:
I agree with you. I need some way in these tests to get around the random ordering of bidders to fit into the json test framework. I could pull this out into a separate unit test and not need to make this change. I also feel it's probably ok in this case since this isn't a literal "if test, do this" code block and there are potential use cases where an bid generator might want to generate bidder specific ids.